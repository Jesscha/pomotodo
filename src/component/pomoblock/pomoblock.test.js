import React from 'react';
import { shallow } from 'enzyme';
import Pomoblock from './pomoblock.component';

describe("<Pomoblock />", ()=>{
    const wrapper = shallow(<Pomoblock />)
   


    it("renders Pomoblock", ()=>{
        expect(wrapper).toMatchSnapshot();
    });

    

    
    })





