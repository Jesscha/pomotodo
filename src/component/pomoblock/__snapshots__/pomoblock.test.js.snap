// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pomoblock /> renders Pomoblock 1`] = `ShallowWrapper {}`;
